src/components/Slider/__tests__/__snapshots__/slider-non-ranged-tests.jest.js.snap

```diff
@@ -56,12 +56,12 @@ Snapshot Diff:
 +           class="monday-style-dialog-content-component top edge-center has-tooltip monday-style-animation-expand-appear-done monday-style-animation-expand-enter-done"
 +         >
 +           <div
-+             class="monday-style-tooltip monday-style-tooltip-dark padding-size-undefined"
++             class="mondayStyleTooltip monday-style-tooltip mondayStyleTooltipDark monday-style-tooltip-dark paddingSizeUndefined padding-size-undefined"
 +           >
 +             20%
 +           </div>
 +           <div
-+             class="monday-style-tooltip-arrow monday-style-arrow monday-style-arrow-dark padding-size-undefined"
++             class="monday-style-tooltip-arrow mondayStyleArrow mondayStyleArrowDark paddingSizeUndefined monday-style-arrow monday-style-arrow-dark padding-size-undefined"
```

Comment:
I understand it is a script but I think it's important to keep the class naming according to our conventions (aka without monday prefix)

Reply:
I'll try to replace it via script

Reply:
Replaced it